tests/databases/test_pymongo.py

```diff
@@ -140,7 +142,7 @@ def test_database_total_score_eleven(data_single_filter_logical_company_code, da
     assert total_score == 11
 
 
-'''
+@pytest.mark.skip(reason="Cannot mock random sampling")
```

Comment:
The test can be fixed? If yes, how?
If this will never work, then we can delete it instead of skipping.

Reply:
Once the issue https://github.com/SAP/odfuzz/issues/9 has been resolved, we will need this test case.

Reply:
Thank you, this is the info I needed.

Reply:
I have added it directly to the code for easier reference than finding out trough necessary search  relevant commit and PR/Issues